ngraph/core/include/ngraph/op/hswish.hpp

```diff
@@ -0,0 +1,54 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/util/unary_elementwise_arithmetic.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v4
+        {
+            /// \brief A HSwish Activation Function
+            /// f(x) =  x * min(max(x + 3, 0), 6) / 6 or
+            /// f(x) = x * min(ReLU(x + 3), 6) / 6
+            ///
+            class NGRAPH_API HSwish : public ngraph::op::util::UnaryElementwiseArithmetic
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"HSwish", 4};
```

Comment:
Please use NGRAPH_RTTI_DECLARATION/DEFINITION

Reply:
In hpp file: `NGRAPH_RTTI_DECLARATION;`
In cpp file: `NGRAPH_RTTI_DEFINITION(op::v4::HSwish , "HSwish", 4);`

Reply:
Done